getnotes/getnotes.py

```diff
@@ -414,8 +422,65 @@ async def findplayer(self, ctx, *,player: Union[ipaddress.IPv4Address, int, str]
         
         except ModuleNotFoundError:
             return await message.edit(content="`mysql-connector` requirement not found! Please install this requirement using `pip install mysql-connector`.")
-                      
+
+
+    async def dph(self, ctx) -> dict: # Deaths per hour
+        """
+        Calculate a player's deaths per hour
+        """
+        player = key_to_ckey(player)
```

Comment:
Using player before it's defined. Should be fixed by making it a required arg (see previous comment)